multi-agent/lib/mas/elements/nodes/claude_agent/claude_agent_node.py

```
@@ -0,0 +1,504 @@
+"""
+Claude Agent Node - Runs autonomous Claude Agent SDK sessions
+
+Delegates work to a Claude Agent SDK session configured with
+model, tools, skills, and authentication credentials.
+"""
+
+from typing import Optional, Any, List, ClassVar, Dict
```

Ruff (0.15.15)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)